作者有增加画中画播放的打算吗？

Leexiaolong opened the pull request:
直播和点播如果都能增加后台小窗播放功能就好了

1062930629 commented:
666

lxl217114 commented:
选择【倒序】排序时，上一集与下一集播放顺序也会变成【倒序播放】
希望后续的版本能修复这个bug
【倒序排序】的时候也能【正序播放】

lxl217114 commented:
麻烦作者
有空的时候修复一下倒序播放功能
即目前在【倒序显示】的前提下，播放顺序也是【倒序】的。
【倒序显示】（方便找到最后一集），播放的顺序应该是【正序播放】

97381736 commented:
大佬你好，有一个小细节能做一下吗？现在市面上所有版本的TVBOX在添加了其他源后就没法再使用内置源了，除非清空数据或者重装，能否配置历史里面增加一个内置源的选择，这样添加其他源后还可以切换回来。

honghuhu commented on app/src/main/java/com/github/tvbox/osc/util/XWalkUtils.java:
```diff
@@ -29,7 +29,8 @@ public interface XWalkState {
     public static String downUrl() {
-        return String.format("https://download.01.org/crosswalk/releases/crosswalk/android/stable/23.53.589.4/%s/crosswalk-apks-23.53.589.4-%s.zip", getRuntimeAbi(), getRuntimeAbi());
+//        return String.format("https://download.01.org/crosswalk/releases/crosswalk/android/stable/23.53.589.4/%s/crosswalk-apks-23.53.589.4-%s.zip", getRuntimeAbi(), getRuntimeAbi());
+        return String.format("http://home.jundie.top:81/xwalk/maven2/crosswalk-apks-23.53.589.4-%s.zip", getRuntimeAbi(), getRuntimeAbi());
```
这里后边传入一个格式化参数就可

lxl217114 commented:
作者有在全屏播放状态下增加长按左侧【快退】长按右侧快进多少秒的设定么？

lwr001ss commented on settings.gradle:
```diff
@@ -1,3 +1,4 @@
-include ':player'
\ No newline at end of file
+include ':player'
+include ':quickjs'
```
```suggestion
include ':quickjs'
```

Yuan-LR commented:
播放器不行啊，解码差不优化吗？

watson1982 committed:
Update activity_collect.xml

watson1982 committed:
Update activity_history.xml

watson1982 committed:
dialog_confirm.xml

watson1982 committed:
Update colors.xml

watson1982 committed:
dialog_confirm.xml

watson1982 committed:
HistoryAdapter

watson1982 committed:
HomeHotVodAdapter

watson1982 committed:
item_grid.xml

watson1982 committed:
item_user_hot_vod

watson1982 committed:
shape_user_delete.xml

watson1982 committed:
Update colors.xml

watson1982 committed:
HomeActivity+del mode

watson1982 committed:
Merge branch 'q215613905:main' into main

q215613905 committed:
紧急修正部分dash视频闪退

watson1982 committed:
UserFragment+check del mode

watson1982 committed:
UserFragment.java

watson1982 committed:
Merge branch 'q215613905:main' into main

watson1982 committed:
Update button_dialog_vod.xml

watson1982 committed:
Update dialog_confirm.xml

watson1982 committed:
Update colors.xml

watson1982 committed:
UserFragment.java

watson1982 committed:
VodController.java

watson1982 committed:
ScreenUtils.java

watson1982 committed:
HomeActivity.java

watson1982 committed:
Update HomeActivity.java

watson1982 committed:
VodController.java

watson1982 committed:
UserFragment.java

watson1982 committed:
icon_lock.xml

watson1982 committed:
icon_unlock.xml

q215613905 committed:
Merge pull request #101 from zhixc/dev

merge By zhixc

q215613905 committed:
Merge pull request #102 from watson1982/main

merge by watson

heudxk commented:
切换   Json并发   Json聚合  Web 聚合  Json轮询  时闪退

670637080 committed:
Update PlayActivity.java

670637080 committed:
Update PlayFragment.java

Commit:
补充修复json聚合并发轮询引起的闪退bug

q215613905 committed:
Merge pull request #105 from 670637080/main-3

Update PlayFragment.java

q215613905 committed:
Merge pull request #106 from 670637080/main-2

merge by 670637080

q215613905 committed:
Merge pull request #107 from zhixc/dev

补充修复json聚合并发轮询引起的闪退bug   merge by zhixc

q215613905 committed:
解决合并打包异常

q215613905 committed:
ijk内置字幕有中文取中文没中文取第一个;解决内置字幕不消失的问题;修磁力不完全解析的问题

watson1982 committed:
Js (#1)

watson1982 committed:
Update build.gradle

q215613905 committed:
Merge pull request #111 from watson1982/main

update quickjs merge by waston

q215613905 committed:
Revert "Update build.gradle"

This reverts commit 2ac956396c937c58d5e26e29e2345800deaddb3f.

q215613905 committed:
Revert "Js (#1)"

This reverts commit 86893e4f2a4c084c67d506b2e0ef88822ddd8da8.

q215613905 committed:
修正一些bug;优化图片加载;(可能是最后一版对4.4以下的支持)

q215613905 committed:
修正

q215613905 committed:
补充提交

q215613905 committed:
支持cronet

q215613905 committed:
本地代理支持转发header

q215613905 committed:
兼容加载base64图片;简易修正个别直播配置加载失败的问题

q215613905 committed:
优化直播

q215613905 committed:
直播配置支持读取自定义hosts;尝试优化ijk

q215613905 committed:
再次优化EXO的dns解析;解决频繁切换导致播放失败的问题

q215613905 committed:
修bug

q215613905 committed:
解决itv直播配置偶现不能能播放的问题

q215613905 committed:
自动去bom头 直播增加壳子代理/proxy?go=live;修正模拟器播放中断的问题

q215613905 committed:
优化播放重试

q215613905 committed:
优化播放逻辑,减少接口调用

q215613905 committed:
暂时去除base64图片兼容

q215613905 committed:
修正exo点播异常;搜索分词修复;设置添加历史配置;支持读取用户自定义doh{"spider":"","doh":[{"name":"Google","url":"https://dns.google/dns-query","ips":["8.8.4.4","8.8.8.8"]}]}

q215613905 committed:
fix bug

q215613905 committed:
直播配置支持.m3u;尝试解决打包权限问题;恢复base64图片

q215613905 committed:
Update build.gradle

q215613905 committed:
change gradle

q215613905 committed:
Update gradle.properties

q215613905 committed:
Update gradle.properties

q215613905 committed:
fix部分爬虫导致的播放器崩溃

yangyamg commented:
🆘嗅探webview组件下载还没有修复 坏了好久你能下载了，麻烦作者大大有空的时候修复一下

🎉请求新功能
🌟直播里能不能加一个“换源”选项

q215613905 committed:
遥控上键刷新,长按菜单进入设置;优化扫码后进入对应页面;

q215613905 committed:
重试逻辑优化(自动切播放器,都失败后切换回原播放器);回车搜索和软键盘确定直接搜索;

q215613905 committed:
直播菜单多源切换;

q215613905 committed:
gradle restore;

q215613905 committed:
优化api设置;修多直播源切换时偶现闪退主页的bug;兼容本地file://配置文件

q215613905 committed:
图片圆角问题修复;优化直播菜单加载逻辑;EPG信息修正;

q215613905 committed:
ijk播放器优化;

q215613905 committed:
增加去广告开关(默认未打开);其他细节调整优化;

q215613905 committed:
修播放重试bug

q215613905 committed:
update gradle

q215613905 committed:
直播配置独立设置(兼容直接传入.txt,.m3u的直播地址);优化分类筛选即时生效

q215613905 committed:
fix bug

q215613905 committed:
fix live setting bug

q215613905 committed:
fix live setting bug

q215613905 committed:
分类页图片不存在时加载纯色背景;修正直播历史当前直播配置对不上的bug

q215613905 committed:
尝试修正播放久了崩溃的问题;

q215613905 committed:
ijk默认选中第一个音轨,解决国粤双语时优先选择粤语的bug;

q215613905 committed:
fix epg底部信息样式不对称问题

q215613905 committed:
解决扫码设置配置地址时直播配置未变动的问题;
解决因直播频道过多导致的频道列表卡顿的问题;

q215613905 committed:
优化配置加载逻辑;
支持仅设置直播地址的情况;

q215613905 committed:
Update test.yml

q215613905 committed:
Update test.yml

q215613905 committed:
Add files via upload

q215613905 committed:
解决直播右上角乱闪
优化epg匹配逻辑
还原正常打包gradle

q215613905 committed:
优化jar加载;

q215613905 committed:
修改进度条样式
点播增加屏显开关
直播支持传递ua参数
直播支持回看

q215613905 committed:
修直播频道列表定时隐藏bug

q215613905 committed:
当前选中直播加载错误时自动切换到下一个

q215613905 committed:
优化屏显;
选集全屏逻辑优化;

q215613905 committed:
修正点播进度条部分设备显示不全的问题;
设置界面新增: 下次进入 (点播|直播);
直播兼容logo图标配置(当配置为false时强制关闭台标显示);
去除加载jar成功的提示;
修正sites配置的filterable站点过滤参数不生效的问题;

670637080 committed:
仅供逻辑参考

q215613905 committed:
修改一处去广告bug;

q215613905 committed:
fix bug

q215613905 committed:
修正部分b战源不能播放的问题;
修正部分直播爬虫分类页异常的问题;

q215613905 committed:
切换搜索站点时减少主页等待时间

q215613905 committed:
fix bug

q215613905 committed:
热词只在第一次打开时加载一次;

q215613905 committed:
支持搜索历史;

q215613905 committed:
直播遥控左右键切换线路;
直播支持数字键换台;

q215613905 committed:
fix bug

q215613905 committed:
fix bug;

q215613905 committed:
替换加载出错的占位图;

q215613905 committed:
优化指定搜索源逻辑;
fix live bug;

q215613905 committed:
数据备份可删除

q215613905 committed:
fix bug

q215613905 committed:
fix bug

q215613905 committed:
历史记录保留非当前配置源;

q215613905 committed:
fix bug;

q215613905 committed:
首页焦点默认定位到主页

q215613905 committed:
优化主页推荐加载

q215613905 committed:
fix bug;

q215613905 committed:
fix bug;

q215613905 committed:
fix bug;

q215613905 committed:
详情页倒序按钮下移;
详情页新增简介按钮弹窗;

q215613905 committed:
fix 集数大于1就显示倒序;

q215613905 committed:
Revert "fix 集数大于1就显示倒序;"

This reverts commit a977cf6efb790ecc53f6326654460444954eb03d.

q215613905 committed:
fix bug;

q215613905 committed:
修正从后台返回继续播放时黑屏的bug;
fix live bug;

q215613905 committed:
解决直播tv端选中播放焦点丢失的问题;
遥控器没有菜单按键时可长按确定键弹出直播菜单;

q215613905 committed:
fix live channel bug;

q215613905 committed:
Merge pull request #142 from 670637080/patch-2

修搜索bug by 670637080

q215613905 committed:
解决代码漏提导致的部分详情闪退;

q215613905 committed:
fix详情光标bug;

q215613905 committed:
首页站点推荐补全

q215613905 committed:
优化详情简介样式

q215613905 committed:
更改拼音联想接口

q215613905 committed:
优化首页加载时间;
完美版,以后更新随缘;

q215613905 committed:
fix;

q215613905 committed:
;

q215613905 committed:
Create README.md

q215613905 committed:
Update README.md

q215613905 committed:
Update README.md

q215613905 committed:
Update README.md

q215613905 committed:
Update README.md

q215613905 committed:
Update README.md

q215613905 committed:
Update test.yml

q215613905 committed:
fix jar;

q215613905 committed:
优化多jar及jar缓存命中的逻辑;
兼容无jar的纯采集json配置;

q215613905 committed:
优化多jar及jar缓存命中的逻辑;
兼容无jar的纯采集json配置;

q215613905 committed:
优化自动切换播放器的时机;
兼容type3类型的需要独立jar支持的直播;

q215613905 committed:
优化完善嗅探click点击事件

q215613905 committed:
尝试适配“王二小放牛娃”

q215613905 committed:
遥控长按上键倍速播放;

q215613905 committed:
fix last commit

q215613905 committed:
fix t4;
搜索小调;
去除jar加载时的冗余逻辑;

q215613905 committed:
常驻jar缓存,可手动清理(自带md5无需清理);

q215613905 committed:
长时间未手动清理过jar缓存将自动清理(自带md5除外);

q215613905 committed:
内置超级解析;

q215613905 committed:
内置超级解析;

q215613905 committed:
识别jarCache
示例: {"spider":"./spider.jar","jarCache":"false"}
方便调试的时候关闭缓存

q215613905 committed:
提升用户体验

q215613905 committed:
兼容t3 py;

q215613905 committed:
完善python缓存;

q215613905 committed:
尝试兼容py直播(初版)

q215613905 committed:
独立py和java打包版本

q215613905 committed:
py完善

q215613905 committed:
fix bug

q215613905 committed:
fix bug

q215613905 committed:
fix bug

q215613905 committed:
fix bug

q215613905 committed:
再次完善python;
解决直播地址配置时未及时生效的bug;
（以后更新随缘）

q215613905 committed:
fix bug

q215613905 committed:
恢复备份时自动重启;
优化电视遥控快进快退逻辑 更符合预期;
修正偶现的再按一次退出app不生效的问题;

q215613905 committed:
微调

q215613905 committed:
主页加载数据时可按返回键取消加载(避免等待);

q215613905 committed:
遥控上键直接定位到片头;
修正详情及解析列表焦点问题;

q215613905 committed:
fix bug

q215613905 committed:
直播源加载优化;

q215613905 committed:
尝试解决jar频繁提示加载失败;
fix搜索界面软键盘的问题;

q215613905 committed:
fix 重试;

q215613905 committed:
fix bug

q215613905 committed:
fix python bug

q215613905 committed:
fix详情多源切换时焦点问题

q215613905 committed:
fix bug

q215613905 committed:
fix bug

q215613905 committed:
fix bug

q215613905 committed:
优化详情集数过多时的光标合理性

q215613905 committed:
fix bug

q215613905 committed:
fix bug

q215613905 committed:
fix bug

q215613905 committed:
fix bug

q215613905 committed:
修正历史记录光标边框残留问题;
加载jar错误时立马加载上一次的缓存jar;
详情及播放页焦点问题再次优化;
解决类似海贼王超长剧集滚动目标集数失败的问题;
首页多行卡顿性能优化;
解决部分电影去广告误杀的问题;
片头可按遥控上下键微调;
尝试修正线路过多时因自动排序导致加载过慢的问题;

q215613905 committed:
分组焦点不可外移;

okcaptain committed:
Update Proxy

Add header and support response

okcaptain committed:
Merge pull request #144 from okcaptain/main

Update Proxy

q215613905 committed:
optimize detail

q215613905 committed:
fix ad

okcaptain committed:
Update proguard-rules.pro

Add Nano proguard-rules

okcaptain committed:
Merge branch 'q215613905:main' into main

okcaptain committed:
Merge pull request #145 from okcaptain/main

Update proguard-rules

q215613905 committed:
优化t4 extend;
尝试兼容站点style配置 {"ratio": 1.3330,"type": "rect"};

q215613905 committed:
fix bug

q215613905 committed:
fix style 样式(ratio 0.764 默认竖图 1.333 4:3 1.777 横图)

q215613905 committed:
fix cornerRadius

q215613905 committed:
fix HomeHots

q215613905 committed:
fix bug

q215613905 committed:
修正小窗播放时标题溢出

q215613905 committed:
修正小窗播放时标题溢出

q215613905 committed:
分类筛选点击空白处时隐藏

q215613905 committed:
优化超级解析(同时进行嗅探和并发json);

q215613905 committed:
fix上次提交引发的TV端筛选bug

q215613905 committed:
fix首页站点缓存

q215613905 committed:
fix bug

q215613905 committed:
fix bug;

q215613905 committed:
fix bug

q215613905 committed:
fix bug

q215613905 committed:
fix 后台返回继续播放;
fix 聚搜;

q215613905 committed:
fix 退出应用

q215613905 committed:
fix bug

q215613905 committed:
fix bug;

q215613905 committed:
fix bug;

q215613905 committed:
修正详情分组焦点切换线路时的bug;

q215613905 committed:
新增简化去广告rules写法{"name":"非凡","hosts":["vip.ffzy","hd.ffzy"],"regex":["6.666667","6.566667","2.966667","2.433333","-2.266667"]} (正数代表移除第一次出现的切片组,负数代表最后一次出现的切片组);
增加去除广告条数统计提示

q215613905 committed:
fix AdCounts;

q215613905 committed:
fix AdCounts;

q215613905 committed:
fix AdCounts;

q215613905 committed:
升级quikjs;
优化JsLoader;

q215613905 committed:
fix bug

q215613905 committed:
优化默认js/lib文件加载;
优化广告规则切片匹配;

q215613905 committed:
优化

q215613905 committed:
规则rules配置兼容script{"name": "剧圈圈","hosts": ["jqqzx"],"script": ["document.getElementById('playleft').children[0].contentWindow.document.getElementById('start').click()"]}
优化搜索历史;

q215613905 committed:
fix

q215613905 committed:
fix js模块加载问题;
解决部分drpy遗留问题;

q215613905 committed:
fix 指定直播playerType的bug;

q215613905 committed:
fix t4;

q215613905 committed:
详情页显示实际播放地址;

q215613905 committed:
解决部分盒子不能正常筛选分类的问题;

q215613905 committed:
优化屏显样式;

q215613905 committed:
fix bug;

q215613905 committed:
exo支持音轨切换

q215613905 committed:
fix bug;

q215613905 committed:
音轨记忆;
修正自动切换播放器失效;

q215613905 committed:
fix bug;

q215613905 committed:
fix bug;

q215613905 committed:
修正doh未生效的问题;
快进快退更激进;

q215613905 committed:
fix style;fix 嗅探;

q215613905 committed:
修正部分源历史记录图片不显示的问题;
fix style;

q215613905 committed:
修正一处去广告的bug

yangyamg commented:
希望直播能支持7天回看 ，在节目单前面加上周一至周日

robinlubin commented:
可以增加调用当贝播放器吗，感谢

q215613905 committed:
修一个t4 ext参数相关的bug;更改一处盒子/手机的判断条件

q215613905 committed:
修一个t4 ext参数相关的bug;更改一处盒子/手机的判断条件

q215613905 committed:
Merge remote-tracking branch 'origin/main'

q215613905 committed:
修正部分本地jar代理直播失败的bug

q215613905 committed:
修正type0直播不能走jar代理的bug

q215613905 committed:
最低支持安卓4.4+
优化及修正

q215613905 committed:
音轨选择优化
修正播放问题

q215613905 committed:
直播增加ijk/exo自动切换

q215613905 committed:
增加64位打包
修复配置切换异常

q215613905 committed:
优化及修正python

q215613905 committed:
优化直播界面

q215613905 committed:
自动切换线路
python拆分32/64

q215613905 committed:
直播切台优化

q215613905 committed:
优化切换线路

q215613905 committed:
直播epg兼容diyp类接口

q215613905 committed:
兼容多仓配置

q215613905 committed:
优化直播焦点问题

q215613905 committed:
自动换线逻辑优化
增加换线开关

q215613905 committed:
优化搜索
可视化搜索进度

q215613905 committed:
Merge remote-tracking branch 'origin/main'

q215613905 committed:
fix bug

q215613905 committed:
优化搜索界面样式

q215613905 committed:
小窗预览追加圆角

q215613905 committed:
包名修改(旧包名已被部分厂家拉黑)

q215613905 committed:
直播显示分辨率
优化流畅度
fix bug

q215613905 committed:
修复搜索导致的闪退及报错

q215613905 committed:
解决个别图床接口解析失败的bug

q215613905 committed:
优化下部分jar的兼容问题
修正下自动换线顺序问题

q215613905 committed:
补齐缺失的jar依赖类及方法

q215613905 committed:
支持danmaku弹幕
主页微调

q215613905 committed:
解决部分设备分类不能筛选

q215613905 committed:
优化python及jar代理问题

q215613905 committed:
设置界面增加弹幕配置
兼容jar的action动作
优化使用体验

q215613905 committed:
其他优化

q215613905 committed:
fix 进度

q215613905 committed:
兼容影视等他壳专用jar

q215613905 committed:
exo容错

q215613905 committed:
去广告增强

q215613905 committed:
去广告增强

q215613905 committed:
解决部分源历史记录异常

q215613905 committed:
去广告增强

q215613905 committed:
配置地址增加本地文件选择
远程可配置直播地址

q215613905 committed:
解决播放时偶发白屏闪退
修正代理端口偶发-1的bug

q215613905 committed:
修正分类数据加载偶发bug

q215613905 committed:
修正代理偶发播放异常

q215613905 committed:
修正搜索偶发闪退

q215613905 committed:
修正播放菜单异形

q215613905 committed:
弹幕api可远程扫码配置

q215613905 committed:
支持弹幕推送http://127.0.0.1:9978/action?do=refresh&type=danmaku&path=

q215613905 committed:
支持logvar弹幕

q215613905 committed:
支持logvar弹幕

q215613905 committed:
优化logvar弹幕匹配

q215613905 committed:
优化js爬虫兼容性(修复各种js导致的闪退)

q215613905 committed:
优化jar加载

q215613905 committed:
直播完整支持m3u|txt

q215613905 committed:
直播支持配置切换

q215613905 committed:
兼容影视壳的Proxy配置规则
修正安全dns

q215613905 committed:
优化直播加载

q215613905 committed:
解决诸如php转发直播等不能播放的bug

q215613905 committed:
修正混淆导致的分类无数据

q215613905 committed:
优化热词加载

q215613905 committed:
优化直播加载

q215613905 committed:
优化直播加载

q215613905 committed:
直播源支持加载js和py的liveContent

q215613905 committed:
py爬虫ext兼容

q215613905 committed:
py优化;支持站点timeout参数

q215613905 committed:
修正特定格式播放失败的bug
优化jar加载

q215613905 committed:
js优化

q215613905 committed:
修直播菜单闪退

q215613905 committed:
部分播放地址播放失败时的兜底播放

q215613905 committed:
修正启动闪退

q215613905 committed:
修正js导致的闪退

q215613905 committed:
修正一处哔哩播放问题

q215613905 committed:
修正一处哔哩播放问题

q215613905 committed:
尝试解决部分模拟器主页报错的问题

JJBJJ committed:
新增弹幕搜索扩展入口

q215613905 committed:
播放header增强
exo调整

q215613905 committed:
播放header增强
exo调整

q215613905 committed:
优化详情封面布局

q215613905 committed:
优化详情封面布局

q215613905 committed:
Merge remote-tracking branch 'origin/main'

q215613905 committed:
Merge remote-tracking branch 'origin/main'

q215613905 committed:
Merge pull request #151 from JJBJJ/feature/danmu-search

新增弹幕搜索扩展入口

q215613905 committed:
修bug

q215613905 committed:
图片加载优化

q215613905 committed:
优化jar和py加载

q215613905 committed:
优化关闭小窗预览时的播放逻辑
解决详情回搜索时偶发的闪退

q215613905 committed:
优化配置加载

q215613905 committed:
合并main

q215613905 committed:
优化站点加载

q215613905 committed:
fix图片

q215613905 committed:
okhttp调整

q215613905 committed:
优化站点切换加载效果

q215613905 committed:
fix直播解析bug

q215613905 committed:
fix bug

q215613905 committed:
详情海报背景优化

q215613905 committed:
聚搜调整

q215613905 committed:
修正几处图片边界问题

q215613905 committed:
主页上键刷新站点数据
其他调整

q215613905 committed:
播放界面锁定返回键小调整

q215613905 committed:
优化手册加载爬虫预热

q215613905 committed:
修正台标显示不全

q215613905 committed:
解决图片偶发的闪退

q215613905 committed:
优化切换配置体验

q215613905 committed:
修正一处自动换线的bug

q215613905 committed:
详情退回时异常兜底

q215613905 committed:
Revert "详情退回时异常兜底"

This reverts commit 23cedebdbe10b05e9d09b8501bb538d23924817e.

q215613905 committed:
修正主页偶发的闪退

q215613905 committed:
修正主页偶发的闪退

q215613905 committed:
python稳定性调整

q215613905 committed:
进度条支持焦点移动

q215613905 committed:
修正手机上偶发样式异常

q215613905 committed:
修正手机上偶发样式异常

q215613905 committed:
增加投屏功能

q215613905 committed:
样式

q215613905 committed:
样式

q215613905 committed:
优化局域网tvbox搜索逻辑

q215613905 committed:
优化局域网dlna搜索逻辑

q215613905 committed:
修正部分投屏设备不能拉起播放器的bug

q215613905 committed:
push_agent兜底

q215613905 committed:
代理地址兼容

q215613905 committed:
播放菜单微调

q215613905 committed:
优化详情线路解析
jar加载过滤壳验证

q215613905 committed:
解决部分手机弹出导航栏的问题

q215613905 committed:
优化python加载

q215613905 committed:
优化js加载

q215613905 committed:
分类顶部下拉刷新

q215613905 committed:
投屏原始地址

q215613905 committed:
优化m3u文本解析

q215613905 committed:
升级quikjs到新架构

q215613905 committed:
Merge branch 'dlna' into quikjs

q215613905 committed:
优化切换配置时的清缓存逻辑

q215613905 committed:
优化投屏header的传递方式

q215613905 committed:
小修

q215613905 committed:
小修

q215613905 committed:
Merge remote-tracking branch 'origin/quikjs' into quikjs

q215613905 committed:
后台预热逻辑优化

q215613905 committed:
解决部分设备推送无反应

q215613905 committed:
播放流程优化

q215613905 committed:
播放流程优化

q215613905 committed:
短视频竖屏播放时可上下滑动切换上一集下一集

q215613905 committed:
支持分辨率切换

q215613905 committed:
修正详情焦点乱闪

q215613905 committed:
修正详情焦点乱闪
播放屏显常驻

q215613905 committed:
小修

q215613905 committed:
小修

q215613905 committed:
小修

q215613905 committed:
小修

q215613905 committed:
详情超时调整

q215613905 committed:
弹幕修正

q215613905 committed:
移除旧quickjs遗留代码

q215613905 committed:
修正python32包部分设备莫名闪退

q215613905 committed:
详情返回时先返回线路焦点

q215613905 committed:
弹幕兜底增强

q215613905 committed:
修正后台返回播放

q215613905 committed:
fix偶发样式异常

q215613905 committed:
fix偶发样式异常

q215613905 committed:
优化配置解析兼容性

q215613905 committed:
详情无数据时自动切换站源

q215613905 committed:
分类海报不再进搜索,直接进详情兜底

q215613905 committed:
修正配置解析相对路径的一处bug

q215613905 committed:
历史记录同剧名只保留最新站点数据

q215613905 committed:
优化切源时的集数匹配

q215613905 committed:
增加同名历史合并开关

q215613905 committed:
修正播放菜单偶发跳变

q215613905 committed:
修正播放菜单偶发跳变

q215613905 committed:
修正首页推荐布局异常

q215613905 committed:
修正打包后分类无数据

q215613905 committed:
时间统一24小时制

q215613905 committed:
修正继续搜索时的计数问题

q215613905 committed:
屏显进度时间显示优化

q215613905 committed:
直播界面微调
直播回看逻辑优化

q215613905 committed:
修直播配置切换时偶发bug

q215613905 committed:
直播界面微调

q215613905 committed:
修正jar一处加载bug

q215613905 committed:
优化字幕搜索;
exo支持内置字幕;
支持视轨选择;

q215613905 committed:
exo增加FFmpeg组件

q215613905 committed:
主页ui微调

q215613905 committed:
字幕微调

q215613905 committed:
字幕微调

q215613905 committed:
支持在线搜索弹幕

q215613905 committed:
尺寸|倍速|弹幕支持长按

q215613905 committed:
上|下一集长按选集;
遥控菜单键选集;

q215613905 committed:
优化直播切台

q215613905 committed:
恢复分类页msearch:支持

q215613905 committed:
详情预览时-遥控菜单键快速切源

q215613905 committed:
样式微调

q215613905 committed:
样式微调

q215613905 committed:
详情展示优化html标签过滤

q215613905 committed:
py兼容性增强

q215613905 committed:
去广告增强

q215613905 committed:
搜索智能排序

q215613905 committed:
搜索智能排序

q215613905 committed:
解决个别站点分类无数据

q215613905 committed:
修正多jar配置搜索偶发崩溃

q215613905 committed:
历史和收藏长按进搜索
详情为空兜底逻辑优化

q215613905 committed:
详情加载优化
修正渲染方式变更未及时生效的bug

q215613905 committed:
搜索微调

q215613905 committed:
修正一处surface渲染的bug

q215613905 committed:
调整默认播放器为exo

q215613905 committed:
直播菜单自动隐藏逻辑优化

q215613905 committed:
弹幕在线搜索弹框焦点修正

q215613905 committed:
修正部分盒子设备应用无法退出的bug
